sorc/sfc_climo_gen.fd/interp.F90

```diff
@@ -259,30 +258,17 @@ subroutine interp(localpet, method, input_file)
 
  end subroutine interp
 
- subroutine adjust_for_landice(field, vegt, idim, jdim, field_ch)
+!> Ensure consistent fields at land ice points
+!! Land ice is vegetation type 15 (variable landice).
+!! @author George Gayno NCEP/EMC
+!! Usage: call adjust_for_landice(field, vegt, idim, jdim, field_ch)
+!! output is Model field
+!! @param field         - Model field before adjustments for land ice.
+!! @param field_ch      - Field name.
+!! @param i/jdim        - i/j dimension of model tile.
+!! @param vegt          - Vegetation type on the model tile.
```

Comment:
author tag for subroutine as well as file.

Reply:
Included.